src/database/initialization/base_field_to_json.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE FUNCTION base_field_to_json(base_field base_fields)
+RETURNS JSONB AS $$
+BEGIN
+  RETURN jsonb_build_object(
+    'id', base_field.id,
+    'label', base_field.label,
+    'description', base_field.description,
+    'shortCode', base_field.short_code,
+    'dataType', base_field.data_type,
+    'createdAt', to_json(base_field.created_at)::jsonb
+  );
+END;
+$$ LANGUAGE plpgsql;
```

Comment:
Does this mean that TinyPG is no longer aware of column/property types (beyond "object") when getting the data, but that type checking would still occur at conversion to the AJV type?

Reply:
I believe that the type is still maintained (as far as JSON has types) -- e.g. a string vs a number vs an object.

We've been moving away from type checking the outputs of the database since we control that layer completely.